src/gurt/dlog.c

```diff
@@ -912,6 +927,16 @@ d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 				mst.log_file, strerror(errno));
 			goto error;
 		}
+		rc = fcntl(mst.log_fd, F_DUPFD, 128);
```

Comment:
it could be good to add a one line comment that explains why you did that. I understand now by reading the PR comment but it may not be as obvious while reading the code.

Reply:
It's a bit obscure so perhaps better to include the ticket number in a comment?